index.js

```diff
@@ -101,9 +101,13 @@ instance.prototype.init = function() {
 			}
 		});
 
-		self.obs.on('SceneCollectionChanged', function() {
```

Comment:
Was the `SceneCollectionChanged` trigger not needed?

Reply:
I thought I had just moved it down, so that the authentication action was organized with the connection actions.

Reply:
Thanks for catching! Should be back now

Reply:
Previously, there were two of these, the 2nd one being further down and consisting of:
```
		self.obs.on('SceneCollectionChanged', (data) => {
			self.updateCurrentSceneCollection()
		})
```

That's a different call then this one, but I'm not sure if these are effectively duplicated in this branch?

Reply:
It looks like there are 2 on the master branch as well, but I'm not entirely sure why. I can look into it. Looks like the second one got added here: https://github.com/bitfocus/companion-module-obs-studio/pull/46/commits/ff8d6d53a63640bcca19a547d512007e9857aec0

Reply:
Yeah, checking for duplicates like that is not something Jeffrey or I would typically scan for.  If you can clean that up/consolidate that would be helpful.  thx!

Reply:
Ok while looking into the duplicate, it looks like  https://github.com/bitfocus/companion-module-obs-studio/commit/ff8d6d53a63640bcca19a547d512007e9857aec0 actually completely broke variables and I'm not sure why.

Reply:
@krocheck Ok thanks for the pointers, it looks like variables (and other things) were being init everywhere but...at init. So I made that change below. Also tested consolidating the duplicated and that seems to work too.

Let me know if anything looks funky here.